_config.yml

```diff
@@ -23,11 +23,16 @@ defaults:
       githubbranch: "v1.7.3"
       docsbranch: "release-1.7"
       versions:
+      - fullversion: "v1.10.0"
+          version: "v1.10"
+          githubbranch: "v1.10.0"
+          docsbranch: "release-1.10"
+          url: https://kubernetes.io/
```

Comment:
This should be https://v1-10.docs.kubernetes.io/  - and 1.11 is released and need to be mentioend here.